tensorboard/backend/event_processing/io_wrapper_test.py

```diff
@@ -64,32 +86,30 @@ def testListDirectoryAbsolute(self):
         (os.path.join(temp_dir, f) for f in expected_files),
         io_wrapper.ListDirectoryAbsolute(temp_dir))
 
-  def testListRecursivelyForNestedFiles(self):
+  def testGlobAndListFiles(self):
```

Comment:
> test the glob method by checking that it returns the exact same data (up to ordering) as the tried-and-true walk method.

Awesome idea. The 2 methods now return via the same format, but their tests are slightly different since `GlobAndListFiles` excludes subdirectories from listings.